src/pages/TimeTracker/styles/TimeTracker.styles.scss

```diff
@@ -0,0 +1,49 @@
+@import "../../../assets/styles/styles.scss";
+
+.createEntrie {
+  background-color: #f8f9fa;
+  border: 1px solid #eaeaea;
+  border-radius: 8px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.1);
+}
+
+.createEntrie .btn-create {
```

Comment:
(suggestion; blocking): This is probably a typo. `createEntrie` should probably be renamed to `createEntry`